tests/testthat/test-dataset.R

```diff
@@ -1,45 +1,50 @@
-context("Dataset object and methods") 
+context("Dataset object and methods")
 
 with(fake.HTTP, {
-    test.ds <- loadDataset("an archived dataset")
+    test.ds <- loadDataset("test ds")
+    today <- Sys.Date()
     test_that("Dataset attributes", {
-        expect_identical(start_date(test.ds), NULL) 
+        expect_identical(start_date(test.ds), NULL)
         expect_identical(end_date(test.ds), NULL)
     })
     test_that("start_date makes right request", {
-        expect_error(start_date(ds, "Today"),
-        'POST /api/datasets/dataset2/start_date.json \\{"start_date":"Today"\\}')
+        expect_error(
+            start_date(test.ds) <- today,
```

Comment:
the first argument to `expect_error` is the code to execute. So for this it’s the `<-` part.